orchestrator/api/memory_stats.py

```
@@ -231,6 +231,138 @@ async def get_recent_memories(
+            "source": "mem0",
+            "search_query": query,
+        }
+    except Exception as e:
+        logger.error("Memory browse failed: %s", e, exc_info=True)
+        return {"success": False, "error": str(e)[:200], "memories": []}
+
+
+@router.delete("/{memory_id}")
+async def delete_memory(
+    memory_id: str,
+    ctx: RequestContext = Depends(get_request_context_hybrid),
```

Ruff (0.15.5)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)